doc/docs/master/deploy-manage/deploy/deploy-w-proxy.md

```diff
@@ -553,3 +553,72 @@ The `pachyderm-proxy` service also routes Pachyderm's [**S3 gateway**](../../man
 **access Pachyderm's repo through the S3 protocol**) on port 80 (note the endpoint in the diagram below).
 
 ![Global S3 Gateway with Proxy](../../images/main-s3-gateway-with-proxy.png)
+
+## Changes to the Enterprise Server Setup
+
+Your enterprise server is [deployed in the same way as any regular cluster](#deploy-pachyderm-in-production-with-a-proxy) with a few differences (no object-store and one unique PostgreSQL database required: `dex`). The same applies when deploying an enterprise server with a proxy. 
```

Comment:
I think we still require the pachyderm database.  https://pachyderm.atlassian.net/browse/FREQQ-70 refers to enterprise *members* not needing a dex database, but the enterprise server still needs to store some data in pachyderm like the license key, in addition to needing dex to serve the auth server.

I verified this by installing with `pachd.enabled=false` and `enterpriseServer.enabled=true` and looking at the database; pachd (in the `pach-enterprise-...` pod) applies the migrations, and it writes to tables like `license.clusters`.

Reply:
Yes, you are correct https://pachyderm.slack.com/archives/CDWDPKPMY/p1658496327199979

Made the change to request the 2 databases for the Standalone Enterprise server.